### PR TITLE
Refactor navigation framework from React Navigation to Bare React

### PR DESCRIPTION
**Pull request by NathHorrigan**

### CODE REVIEW

File: src/containers/ExperimentContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    terminateExperiment(false)
+    return null
+  }
+
+  // Get the current module data and it's respective component.
+  const ModuleComponent = ExperimentModuleTypes[currentModule?.moduleType]
+
+  // This function is called when the experiment should transition to next module.
+  function onModuleComplete() {
+    dispatch(incrementExperimentModule())
```
Comment (mixxorz) [inline src/containers/ExperimentContainer.tsx]: Suggestion: maybe call this action `nextModule()`?

File: src/screens/LoginScreen.tsx [hunk 2]
```diff
@@ -4,14 +4,18 @@ import Spinner from 'react-native-spinkit'
 import Config from 'react-native-config'
 import camelcaseKeys from 'camelcase-keys'
 
-import { ModuleScreen } from '@screens'
 import { Box, Text, Button, Image, TextField } from '@components'
-import { ExperimentViewController, Experiment } from '@controllers'
+import {
+  ExperimentViewController,
```
Comment (mixxorz) [inline src/screens/LoginScreen.tsx]: Invalid import maybe?

**Review by mixxorz — COMMENTED** [review]
This looks really good. Thanks Nathan.

File: src/containers/ExperimentContainer.tsx (hunk 1 above)
Comment (NathHorrigan) [reply src/containers/ExperimentContainer.tsx]: Good Idea!

**Pull request merged**